plugins/flytekit-papermill/flytekitplugins/papermill/task.py

```diff
@@ -255,6 +259,10 @@ def execute(self, **kwargs) -> Any:
         singleton
         """
         logger.info(f"Hijacking the call for task-type {self.task_type}, to call notebook.")
+        for k, v in kwargs.items():
+            if isinstance(v, (FlyteFile, FlyteDirectory, StructuredDataset)):
```

Comment:
nit: store this in a global variable in the top-level scope of this module e.g.

```
SAVE_TO_LITERAL = (FlyteFile, FlyteDirectory, StructuredDataset)
```

It's effectively the same, but will be easier to extend this to perhaps other types we want to save to literal.